src/Console/Commands/AuthMigration.php

```diff
@@ -25,7 +26,7 @@ public function __construct(Composer $composer)
     public function handle()
     {
         $from = __DIR__.'/stubs/auth/statamic_auth_tables.php.stub';
-        $file = date('Y_m_d_His', $this->option('test') ? 1648622397 : time()).'_statamic_auth_tables';
+        $file = Carbon::now()->format('Y_m_d_His').'_statamic_auth_tables';
```

Comment:
The test param was needed otherwise tests were failing for me (locally and on GitHub) and it meant migrations needed constantly cleared.

I dont see anything in your changes that have worked around that.

Can you explain your logic please?

Reply:
Using `Carbon::setTestNow()` in tests effectively handles this case, and you already did that